agroal-pool/src/main/java/io/agroal/pool/ConnectionPool.java

```diff
@@ -581,16 +582,11 @@ private final class FlushTask implements Runnable {
 
         @Override
         public void run() {
-            if ( handler != null ) {
+            for ( ConnectionHandler handler : handler != null ? Collections.singleton( handler ) : allConnections ) {
```

Comment:
I would do

```
Collection handlers = null;
if (handler != null) {
  handlers = Collections.singleton(handler);
} else {
  handlers = allConnections;
}
for (ConnectionHandler ch : handlers) {
```

Having `ConnectionHandler handler : ` as a variable shadow is confusing

Reply:
renamed the variable to remove shadowing